src/core/common/schemas.ts

```diff
@@ -11,11 +11,3 @@ export const adapterParamsSchema = {
 };
 
 export type AdapterParams = { adapter: string };
-
-export const fileBodySchema = {
```

Comment:
If we gonna send post request without file what will do?

Reply:
I've added validation for that